### ISSUE TITLE
What to do with the scientific-python / nightly-wheels Public

### ISSUE THREAD

**Issue by jarrodmillman** [scientific-python/upload-nightly-action#5]
Should we close this https://github.com/scientific-python/upload-nightly-action/issues/6 and then delete the old repo?

**Comment by matthewfeickert** [comment]
Ah I had missed that @Carreau had made this and finished what I hadn't. Great! @jarrodmillman maybe archive things but don't delete right away so that if there is information in those issues that should get checked against here we can still have them?

**Comment by bsipocz** [comment]
Let me rephrase. This action was implemented on the first day of the summit and has been applied to several repos. At this point, I don't think there is any point of pulling in something different that none of us is familiar with.

Removing the other repo would make sense IMO, otherwise it would introduce some confusion into the system.

**Comment by matthewfeickert** [comment]
> This action was implemented on the first day of the summit and has been applied to several repos. At this point, I don't think there is any point of pulling in something different that none of us is familiar with.

I'm at work now, and I wasn't invited to the summit, so I'll have to check back later but I believe that there are things that were discussed in the now deleted repo that aren't implemented here that came up as use issues. I'll check my emails and [the related Discourse topic](https://discuss.scientific-python.org/t/interest-in-github-action-for-scipy-wheels-nightly-uploads-and-removals/397) and see.

**Comment by bsipocz** [comment]
> were discussed in the now deleted repo

In my recollection that repo was empty with the exception of a licence file.

**Comment by matthewfeickert** [comment]
> > were discussed in the now deleted repo
> 
> In my recollection that repo was empty with the exception of a licence file.

It was not. You didn't look at the other branches.

**Comment by Carreau** [comment]
Sorry for the confusion and deletion if there was any. I agree that we should not delete but use the GitHub archiving feature when possible.

**Comment by bsipocz** [comment]
> GitHub archiving feature

there has been an empty repo with a licence file in it. If unfinished development happened in branches, rather than on works, without even a draft PR opened, that is unfortunate, but still doesn't necessitate the archiving and introduction of possible confusion into the system.
